app/code/Magento/Review/Model/Rating/Entity.php

```diff
@@ -28,7 +31,7 @@ protected function _construct()
      * @param string $entityCode
      * @return int
      */
-    public function getIdByCode($entityCode)
+    public function getIdByCode(string $entityCode): int
```

Comment:
Please see the actual implementation - the return value is retrieved using \Magento\Framework\DB\Adapter\AdapterInterface::fetchOne that actually returns a string - so the declared return strict type results in a fatal error.

Please ensure strict types are matching the actual return values